tests/test_rocm_sanity.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import subprocess
+import re
+from pathlib import Path
+
+THIS_DIR = Path(__file__).parent
```

Comment:
From https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parent
> If you want to walk an arbitrary filesystem path upwards, it is recommended to first call [Path.resolve()](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve) so as to resolve symlinks and eliminate ".." components.

```suggestion
THIS_DIR = Path(__file__).resolve().parent
```